plugins/translator/src/main/java/org/jivesoftware/spark/translator/TranslatorPlugin.java

```diff
@@ -81,7 +81,6 @@ public void filterOutgoing(ChatRoom room, Message message) {
                             String currentBody = message.getBody();
                             Language lang = (Language) translatorBox.getSelectedItem();
                             if (lang != null && lang != Language.NONE) {
-                                message.setBody(null);
```

Comment:
Good catch! This should be recorded in a ticket. I create a new ticket for this: https://igniterealtime.atlassian.net/browse/SPARK-2374 - please change the commit message to include a reference to SPARK-2374 so that automation will link the code change with the ticket.